conn.go

```diff
@@ -591,6 +591,8 @@ func (cn *conn) simpleExec(q string) (res driver.Result, commandTag string, err
 	b.string(q)
 	cn.send(b)
 
+	res = emptyRows
```

Comment:
This isn't really correct.  This should only be assigned after EmptyQueryResponse ('I').